zenoh/src/api/session.rs

```diff
@@ -2849,7 +2885,7 @@ where
 /// let session = zenoh::open(zenoh::Config::default()).await.unwrap();
 /// # }
 /// ```
-#[must_use = "Resolvables do nothing unless you resolve them using `.await` or `zenoh::Wait::wait`"]
+#[must_use = "Resolvables do nothing unless you resolve them using the `res` method from either `SyncResolve` or `AsyncResolve`"]
```

Comment:
These changes are old. Please make sure you are working on latest `main`.

Reply:
Yep I reverted those.